pydm/widgets/related_display_button.py

```diff
@@ -266,24 +270,41 @@ def open_display(self, filename, macro_string="", target=None):
         # Check for None and ""
         if not filename:
             return
-        
-        macros = parse_macro_string(macro_string)
-        base_macros = find_base_macros(self)
-        merged_macros = base_macros.copy()
-        merged_macros.update(macros)
-        
+
+        parent_display = self.find_parent_display()
+        base_path = ""
+        if parent_display:
+            base_path = os.path.dirname(parent_display.loaded_file())
+
+        fname = find_file(filename, base_path=base_path)
+
+        macros = {}
+        parent_display = self.find_parent_display()
```

Comment:
Sounds like duplicate work? You are already getting it above.

Reply:
Good catch!

Reply:
Addressed via b81f487